ocs_ci/utility/ibmcloud_bm.py

```diff
@@ -167,3 +167,22 @@ def restart_machines_by_stop_and_start(self, machines):
         """
         self.stop_machines(machines)
         self.start_machines(machines)
+
+    def check_if_machines_are_off_from_sensor_data(self, machines):
```

Comment:
I suggest to rename it to something like filter_active_machines(). From the name of the function it is not clear that it returns list of machines.

Reply:
Renamed the method to understand what the it returns.
Still, I would like to distinguish  between "ON" and "not OFF". The function is not recommended to identify the machines that are "ON".